recipes/ccache/all/conanfile.py

```diff
@@ -26,42 +30,48 @@ class CcacheConan(ConanFile):
         "redis_storage_backend": True,
     }
 
-    generators = "cmake", "cmake_find_package_multi"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     @property
     def _is_msvc(self):
         return str(self.settings.compiler) in ["Visual Studio", "msvc"]
 
```

Comment:
```suggestion
```
Replaced by https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc